x/autopilot/types/keys.go

```diff
@@ -13,3 +15,21 @@ const (
 	// QuerierRoute defines the module's query routing key
 	QuerierRoute = ModuleName
 )
+
+var (
+	TransferFallbackAddressPrefix = []byte("fallback")
+
+	FallbackAddressChannelPrefixLength int = 16
+)
+
+// Builds the store key prefix for a fallback address, key'd by channel ID and sequence number
+// The serialized channelId is set to a fixed array size to assist deserialization
+func GetTransferFallbackAddressKeyPrefix(channelId string, sequenceNumber uint64) []byte {
+	channelIdBz := make([]byte, FallbackAddressChannelPrefixLength)
+	copy(channelIdBz[:], channelId)
+
+	sequenceNumberBz := make([]byte, 8)
```

Comment:
Nit: what's informing choices around the size of this bytearr?

Reply:
It's arbitrary, but that's how I've always seen ints serialized in the SDK